### ISSUE TITLE
Add feature showing data per milion of people

### ISSUE THREAD

**Issue by yami12376** [CSSEGISandData/COVID-19#1963]
It would be good if you would show also data like

> Confirmed: 3 503
> Deaths: 73
> Recovered: 116 
> Active: 3 314

per 1 000 000 people cause it depends how large the country is.
Also it would be helpfull to see daily changes per dot, not only per world.

**Comment by cipriancraciun** [comment]
Others have already proposed something similar to this: #1807, #1791 (see at the end), #975, #1296 

Anyway, I think that JHU will focus on providing the raw data (which is already a big task), and any other augmentations are left to developers (that use this dataset).

That being said, I have derived the JHU dataset and added a few augmentations, among which the per population metrics;  I've described the derived dataset in #1281 and it is available at https://github.com/cipriancraciun/covid19-datasets

(In order to keep things tidy, I would suggest following one of the duplicate issues, and close this one.)

**Comment by jjbenes** [comment]
> Anyway, I think that JHU will focus on providing the raw data (which is already a big task), and any other augmentations are left to developers (that use this dataset).

@cipriancraciun Agreed! JHU's time is better spent on getting the data. There are still many open issues on accuracy.
@yami12376 Reading the CSV files isn't too bad. You can use the code at https://github.com/jjbenes/covid19. There are just two files, one that converts the CSVs into Pandas and another one uses the Pandas to do whatever it needs to do, like computing per-capita data.

The [map](https://first-principles.ai/covid-19/map.html) below was created with some additional back-end code.
![Screen Shot 2020-04-05 at 6 17 03 AM](https://user-images.githubusercontent.com/62985029/78499360-17705100-7705-11ea-9005-8a0f20f3453c.png)
